tests/parser_tests/block_format_test.py

```diff
@@ -465,63 +464,80 @@ async def test_inverse(
     #                               file_edits -> llm_message -> file_edits
     # we get back where we started. So this test verifies things we don't necessarily care about like the order of the
     # edits and white space.
-    cwd = Path(os.getcwd())
-    llm_response = dedent(f"""\
+    llm_response = dedent("""\
         I will insert a comment between the first two lines
         and then replace the last line with 'better measure'
-
         Steps: 1. Insert a comment
                2. Replace last line
-
         @@start
-        {{
-            "file": "{(cwd / "test.txt").as_posix() }",
+        {
+            "file": "test.txt",
             "action": "insert",
             "insert-after-line": 1,
             "insert-before-line": 2
-        }}
+        }
         @@code
         # I inserted this comment
         @@end
         @@start
-        {{
-            "file": "{(cwd / "test.txt").as_posix() }",
+        {
+            "file": "test.txt",
             "action": "replace",
             "start-line": 4,
             "end-line": 4
-        }}
+        }
         @@code
         # better measure
         @@end
         @@start
-        {{
-            "file": "{(cwd / "delete.txt").as_posix() }",
+        {
+            "file": "delete.txt",
             "action": "delete",
             "start-line": 2,
             "end-line": 3
-        }}
+        }
         @@end
         @@start
-        {{
-            "file": "{(cwd / "create.txt").as_posix() }",
+        {
+            "file": "create.txt",
             "action": "create-file"
-        }}
+        }
         @@code
         # I created this file
         @@end
         @@start
-        {{
-            "file": "{(cwd / "file1.txt").as_posix() }",
+        {
+            "file": "file1.txt",
             "action": "rename-file",
-            "name": "{(cwd / "file2.txt").as_posix() }"
-        }}
+            "name": "file2.txt"
+        }
+        @@end
+        @@start
+        {
+            "file": "file1.txt",
```

Comment:
As an example of the rename edge case I mentioned, normally the model would output file2.txt here and we would convert it to file1.txt. If you do end up fixing that edge case (which I'm perfectly fine with you not fixing it), you would change these 2 changes to have file file2.txt.